### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by JGarcia-Panach**
Travis configuration for transifex.
Following the template in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml
and catching the secure from OCA/maintainer-quality-tools#194
Thanks @pedrobaeza

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,36 @@ env:
   - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
   - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
   - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  # This line contains the encrypted transifex password
```
Comment (pedrobaeza) [inline .travis.yml]: You don't need to copy all comment lines. They are there for indicating what to do, but not to be on each final file.

File: .travis.yml [hunk 2]
```diff
@@ -17,6 +17,36 @@ env:
   - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
   - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
   - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
```
Comment (pedrobaeza) [inline .travis.yml:18]: This is not located in the right part

**Comment by pedrobaeza** [comment]
Have you made the commit from other account?

Another thing: please take the occasion to change the coveralls call by the new one (travis_after_success or similar)

**Comment by pedrobaeza** [comment]
Thank you for making this!

**Comment by JGarcia-Panach** [comment]
Thanks to you Pedro, first thing in the morning I will check all your comments and try to fix them

**Comment by JGarcia-Panach** [comment]
With your comments @pedrobaeza and checking other OCA .travis files, I think it should be correct now.
I have made the commit with this account (the only I have in github)
Thanks again for all your help

**Comment by pedrobaeza** [comment]
Yeah, it's correct! :+1: 

I'm saying about the GitHub account because of the CLA. Did you sign it, right?

**Comment by JGarcia-Panach** [comment]
I've signed it this morning.
https://github.com/odoo/odoo/pull/10958

Do I also have to sign ICLA.pdf and send to cla AT odoo-community.org? or it is enough with the PR...

**Comment by pedrobaeza** [comment]
The OCA CLA is not the same. You can see here ours: https://odoo-community.org/page/cla

**Comment by JGarcia-Panach** [comment]
I'm doing it right now. Sorry for all the trouble.

**Comment by pedrobaeza** [comment]
No problem. Thankful for seeing you contributing.

**Comment by legalsylvain** [comment]
Hi @JGarcia-Panach,
did you finally send it ? I'd like to merge this PR and I don't see your CLA in OCA database.

kind regards.

**Comment by JGarcia-Panach** [comment]
Yes I sent it, @legalsylvain

If there is any problem let me know.

Thanks!!

**Comment by atchuthan** [comment]
:+1:

**Pull request merged**